docs/excel/excel-data-types-add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Add properties to basic cell values
+description: Add properties to basic cell values.
+ms.topic: how-to #Required; leave this attribute/value as-is
+ms.date: 04/14/2025
+ms.localizationpriority: medium
+---
+
+# Add properties to basic cell values
```

Comment:
Should this article be tagged as preview? Should any of the other content in this PR (some of the entity card content?) be tagged as preview?

Reply:
If it's not available to all, then yes it should be tagged preview.

Reply:
fixed